bitnami/apache/values.yaml

```diff
@@ -38,6 +38,7 @@ git:
   repository: bitnami/git
   tag: 2.28.0-debian-10-r6
   pullPolicy: IfNotPresent
+  resources: {}
```

Comment:
I think this is not the right place to put this line.
Shouldn't it be at in the block of line 78 ?
```
cloneHtdocsFromGit:
  enabled: false
  # repository:
  # branch:
  interval: 60
```

Reply:
yes your are right. I update the PR.